### PR TITLE
FTP: Add for FTPS the ability to set KeyManager and TrustManager

### PR DESCRIPTION
**Pull request by TheDeadOne**
This fix adds a couple of methods to FtpsSettings to set specific KeyManager and TrustManager.

I understand that the old tag is taken as a basis, but it would be nice to release, for example, version 2.1 with this improvement.

### CODE REVIEW

File: ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala [hunk 1]
```diff
@@ -221,7 +231,9 @@ final class FtpsSettings private (
     s"binary=$binary," +
     s"passiveMode=$passiveMode," +
     s"configureConnection=$configureConnection," +
-    s"proxy=$proxy)"
+    s"proxy=$proxy" +
+    s"keyManager=$keyManager" +
```
Comment (ennru) [inline ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala:235]:
```suggestion
    s"proxy=$proxy," +
    s"keyManager=$keyManager," +
```

**Review by ennru — COMMENTED** [review]
Thank you for suggesting, this addition could go into Alpakka 6.0.4.
Please rebase it.

**Comment by TheDeadOne** [comment]
@ennru I hope everything is done right.

**Comment by ennru** [comment]
One more thing...
```
[error] akka-stream-alpakka-ftp: Failed binary compatibility check against com.lightbend.akka:akka-stream-alpakka-ftp_2.13:6.0.2! Found 1 potential problems
[error]  * method this(java.net.InetAddress,Int,akka.stream.alpakka.ftp.FtpCredentials,Boolean,Boolean,scala.Function1,scala.Option)Unit in class akka.stream.alpakka.ftp.FtpsSettings does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.alpakka.ftp.FtpsSettings.this")
```
It is fine to add an exclusion as the constructor is actually private but Mima doesn't see that.

The filter should go into a file in `ftp/src/main/mima-filters/6.0.3.backwards.excludes`.

**Review by ennru — APPROVED** [review]
LGTM.
Thank you!

**Pull request merged**